_posts/2014-11-13-cockpit-plugin-tutorial.md

```diff
@@ -82,6 +82,9 @@ The pinger tool itself looks like this:
 
 Lets take a look at the pinger HTML, and see how it works.
 
+_**Warning**: Since Cockpit 273 ([relevant pull request](https://github.com/cockpit-project/cockpit/pull/17486)), `cockpit.css` can no longer be included as described below, since it was removed from Cockpit._
```

Comment:
This already got deprecated two years ago in [cockpit 221](https://cockpit-project.org/blog/cockpit-221.html). We have only kept the file on RHEL/CentOS 8 for backwards compatibility, but the Fedora/Debian/Ubuntu and RHEL 9 packages didn't have it any more. I think we should rather just update the blog post to drop the usage of cockpit.css, so that it stays useful.